core/capabilities/ccip/common/crcwconfig.go

```diff
@@ -0,0 +1,80 @@
+package common
+
+import (
+	"context"
+	"fmt"
+
+	chainsel "github.com/smartcontractkit/chain-selectors"
+
+	cciptypes "github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+// GetChainReaderParams is a struct that contains the parameters for GetChainReader.
+type GetChainReaderParams struct {
+	Lggr             logger.Logger
+	Relayer          loop.Relayer
+	ChainID          string
+	DestChainID      string
+	HomeChainID      string
+	Ofc              OffChainConfig
+	ChainSelector    cciptypes.ChainSelector
+	RelayChainFamily string
```

Comment:
We should be consistent with `GetChainWriterParams.ChainFamily`

```suggestion
	ChainFamily string
```